tests/forking_agent/test_executor.py

```diff
@@ -165,6 +166,9 @@ async def test_executor_server_dirty_shutdown(mpmanager: MPManager, caplog):
     assert ["aaaa"] == result
     print("Child there")
 
+    process_name = psutil.Process(pid=child1.process.pid).name()
+    assert process_name == "inmanta: executor test - connected"
```

Comment:
I wondering whether we should include the full identity of an executor here, because multiple agents with the same name can exist at the same time. For example when a dry-run is requested for an unreleased version. With this implementation it will be hard to keep them apart.

Reply:
I agree (see description), I just don't see how to make this identity meaningful...

Reply:
it is volatile and not part of the current interfaces, but I kind of like it.

The ranges may compact quite badly, because not every version may reach the executor so it will not be aware of everything. 

I don't think I'm going to add it now, but it is something to keep in mind

Reply:
I like it as well.